web/src/compose_banner.ts

```diff
@@ -158,3 +160,18 @@ export function show_stream_does_not_exist_error(stream_name: string): void {
     // Open stream select dropdown.
     $("#compose_select_recipient_widget").trigger("click");
 }
+
+// Ensure the input has only one single top-level element,
+// and return a JQuery element of it.
+function parse_single_node(
+    html_element: HTMLElement | JQuery.htmlString,
+): JQuery<Node | HTMLElement> {
+    if (typeof html_element === "string") {
+        const nodes = $.parseHTML(html_element.trim());
+        if (nodes.length === 0 || nodes.length > 1) {
+            throw new Error("Input can only contain one top-level element.");
```

Comment:
I think it would be better to use the `blueslip.error` function here. It is intended for "logging events that are definitely caused by a bug," so I believe it's more appropriate in this context. You can find more information about error handling [here](https://zulip.readthedocs.io/en/latest/subsystems/logging.html#blueslip-frontend-error-reporting).

Reply:
Gotcha. 👍